daemon-tests/tests/main/order.rs

```diff
@@ -23,12 +23,11 @@ async fn taker_places_order_and_maker_rejects() {
 
     ensure_null_next_offers(taker.offers_feed()).await.unwrap();
 
+    let symbol = ContractSymbol::BtcUsd;
     maker
-        .set_offer_params(OfferParamsBuilder::new().build())
+        .set_offer_params(OfferParamsBuilder::new(symbol).build())
```

Comment:
🙃 It's a bit confusing to use different names for this.

```suggestion
        .set_offer_params(OfferParamsBuilder::new(contract_symbol).build())
```

Reply:
IMHO it should just be `symbol` all the way 😄the type can be called as such to avoid confusion, but there's no reason for existence of this mouthful of variable.
fair enough though, I'll move it out of this PR :)